<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginaBusqueda</class>
 <widget class="QMainWindow" name="PaginaBusqueda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>571</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(186, 0, 0);</string>
    </property>
    <property name="text">
     <string>Busca tu auto por precio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(236, 236, 236);
background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>Ingresa el precio inicial:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_precio">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>185</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(236, 236, 236);
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Precio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_barato">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(186, 0, 0);
background-color: rgb(51, 51, 51);
color: rgb(186, 0, 0);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Mas barato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_caro">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(186, 0, 0);
background-color: rgb(51, 51, 51);
color: rgb(186, 0, 0);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Mas caro</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_auto">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section { color: rgb(186, 0, 0); background-color: rgb(51, 51, 51); }

QTableWidget {

	color: rgb(186, 0, 0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>209</number>
    </attribute>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Color</string>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 3px;
background-color: rgb(236, 236, 236);
color: rgb(51, 51, 51);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
